modules/osdk-ansible-k8s-module-installing.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * operators/osdk-ansible.adoc
+
+[id='osdk-ansible-k8s-module-installing-{context}']
+= Installing the k8s Ansible module
+
+To install the `k8s` Ansible module on your local workstation:
+
+.Procedure
+
+. Install Ansible 2.6+:
++
+----
+$ sudo dnf install ansible
```

Comment:
I assume if this is using dnf that means they're running Fedora. Should we be writing this as if they use RHEL instead? Or provide both? Not sure what we normally do in these cases.

Reply:
Switching this to `yum`.